libraries/AP_HAL_ChibiOS/hwdef/f303-TempSensor/hwdef-bl.dat

```diff
@@ -0,0 +1,4 @@
+include ../f303-periph/hwdef-bl.inc
+
+
+define CAN_APP_NODE_NAME "org.ardupilot.ap_periph_tempsensor"
```

Comment:
prefer not to have a custom CAN_APP_NODE_NAME. The default name based on the subdirectory is preferable as it makes it easier to find the fw on firmware.ardupilot.org